backend/cmd/cmd.go

```diff
@@ -41,6 +43,23 @@ func NewRootCmd(
 		"migration migrations root directory",
 	)
 
+	idCmd := cmdFactory.NewCommand(cli.CommandConfig{
+		Usage:        "data-id",
+		ShortHelpMsg: "Use user ID to uniquely identify a user",
+		OnExecute: func(cmd *cli.Command, args []string) {
+			kgsConfig := provider.KgsRPCConfig{
+				Hostname: config.KgsHostname,
+				Port:     config.KgsPort,
+			}
+			keyGenBufferSize := provider.KeyGenBufferSize(config.KeyGenBufferSize)
+			dataTool, err := dep.InjectDataTool(dbConfig, dbConnector, keyGenBufferSize, kgsConfig)
+			if err != nil {
+				panic(err)
```

Comment:
```suggestion
fmt.Println(err)
os.Exit(1)
```

Reply:
@arberiii Why not panic? I think it's okey to crash if we running the tool offline.

Reply:
I thought to keep the same error handling through out the function

Reply:
Done.

Reply:
I think we should `fmt.Error` with a `%w` verb.

Reply:
@alldroll Wa are directly printing the error here. 'fmt.Errorf' returns unused result and didn't provide much value in this case. I think it's better to use `fmt.Errorf` inside the application logic.